config/locales/devise/en.yml

```diff
@@ -116,7 +116,7 @@ en:
       max_login_attempts_reached: >
         Your account is temporarily locked because you have entered the
         one-time passcode incorrectly too many times.
-      please_try_again: Please try again in %{time_remaining}.
+      please_try_again: Please try again in <span id="countdown">%{time_remaining}</span>.
```

Comment:
That seems cleaner. I saw markup in another translation file and figured it was OK to put there. 
I can remove the tag from that file as well, if you don't think it's outside the scope of this PR.